### DISCUSSION TITLE
Welcome to TMS-Discussions!

### DISCUSSION THREAD

**Discussion by davidkowalk** [davidkowalk/tabbed_minecraft_scripting#8] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. This is a place to:
  * Ask questions you’re wondering about.
  * Ask for help when you get stuck or need help on how to use a feature of TMS.
  * Share ideas for projects or potential features.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by busterbeam** [comment]
this doesn't use Kivy, it's not dependent on it.  your script uses Tkinter, which comes prebuilt with almost all *nix's systems.  and on windows python utilizes it for it's IDLE.  Don't waste peoples times with a huge dependency like Kivy

**Reply by davidkowalk** [reply]
Make an Issue.
